src/lando/urls.py

```diff
@@ -83,6 +84,14 @@
     ),
 ]
 
+urlpatterns += [
+    path(
+        "api/pulls/<str:repo_name>/<int:number>",
```

Comment:
Github uses `pulls` rather than `pull`. I think we should match it to avoid confusion [though I think lando uses plurals in general otherwise, so it's a matter of deliberately picking what to be consistent with].

Reply:
Ok, the GitHub API uses plural. Let's stick to that!

Reply:
I *think* that's how I had it already/originally? 🤷